src/main/java/org/jabref/logic/net/ProxyRegisterer.java

```diff
@@ -7,14 +7,14 @@ private ProxyRegisterer() {
 
     public static void register(ProxyPreferences proxyPrefs) {
         if (proxyPrefs.isUseProxy()) {
-            // NetworkTabView.java ensures that proxyHostname and proxyPort are not null
```

Comment:
Probably wrong name?

Reply:
Yes. Very strange, apparently the tab was called NetworkTab once, so changing the name back and forth in the process of decisionmaking revealed this comment...